guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -1,17 +1,51 @@
 [id="glossary-of-terms-used-in-{project-context}_{context}"]
 = Glossary of terms used in {Project}
 
+{Project} is a complete lifecycle management tool for physical hosts, virtual machines, and cloud instances.
+Key features include automated host provisioning, configuration management, and content management including patch and errata management.
+You can automate tasks and quickly provision hosts, all through a single unified interface.
+
+This alphabetically ordered glossary provides an overview of {Project} related technical terms.
+
 [[Activation_key]]
-Activation key:: A token for host registration and subscription attachment.
-Activation keys define subscriptions, products, content views, and other parameters to be associated with a newly created host.
+Activation key::
+Activation keys are used by `subscription-manager` to register hosts to {Project}.
+They define content view and lifecycle environment associations, content overrides, system purpose attributes, and other parameters to be associated with a newly created host.
++
+// Reword when multi-CVs is part of Katello
+They are associated to exactly one lifecycle environment and exactly one content view, though this may be a composite content view.
+You can use them on multiple machines and they behave like configuration information rather than traditional software license keys.
+You can also use multiple activation keys with a single host.
+When a host registers using a given key, content is made available to the host.
```

Comment:
```suggestion
When a host registers using a given key, enabled content from {Project} is provided to the host.
```
Not sure how to phrase this, but not all content might be enabled in an AK.

Reply:
@Lennonka I think that nuance is covered by the next sentence which was my suggestion applied. Maybe "certain content"?

```suggestion
When a host registers using a given key, certain content is made available to the host.
```

```
The content that is made available depends on the content in the activation key's content view and lifecycle environment, any content overrides present, any repository-level restrictions such as operating system or architecture, and system purpose attributes such as release version.
```

Reply:
I missed the next line. Yes, that would work.

Reply:
Applied.